Compile of 3.5.0+80 fails on Windows

**Describe the bug**
On Windows (MinGW64) compilation of dt 3.5.0+80 fails; `M_LN10` undefined


**To Reproduce**
1. `cmake --build .`
2.
```
...
[ 29%] Building C object bin/CMakeFiles/lib_darktable.dir/develop/blend_gui.c.obj
In file included from C:/msys64/mingw64/lib/glib-2.0/include/glibconfig.h:9,
                 from C:/msys64/mingw64/include/glib-2.0/glib/gtypes.h:32,
                 from C:/msys64/mingw64/include/glib-2.0/glib/galloca.h:32,
                 from C:/msys64/mingw64/include/glib-2.0/glib.h:30,
                 from C:/msys64/home/lintujuh/darktable/src/common/database.h:21,
                 from C:/msys64/home/lintujuh/darktable/src/common/darktable.h:46,
                 from C:/msys64/home/lintujuh/darktable/src/common/colorspaces_inline_conversions.h:22,
                 from C:/msys64/home/lintujuh/darktable/src/common/iop_profile.h:22,
                 from C:/msys64/home/lintujuh/darktable/src/develop/blend.h:21,
                 from C:/msys64/home/lintujuh/darktable/src/develop/blend_gui.c:18:
C:/msys64/home/lintujuh/darktable/src/develop/blend_gui.c: In function 'log10_scale_callback':
C:/msys64/home/lintujuh/darktable/src/develop/blend_gui.c:716:32: error: 'M_LN10' undeclared (first use in this function); did you mean 'G_LN10'?
  716 |       outval = CLAMP_RANGE(exp(M_LN10 * (4.0f * inval - 4.0f)), 0.0f, 1.0f);
      |                                ^~~~~~
C:/msys64/home/lintujuh/darktable/src/develop/blend_gui.c:716:16: note: in expansion of macro 'CLAMP_RANGE'
  716 |       outval = CLAMP_RANGE(exp(M_LN10 * (4.0f * inval - 4.0f)), 0.0f, 1.0f);
      |                ^~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
make[2]: *** [bin/CMakeFiles/lib_darktable.dir/build.make:1258: bin/CMakeFiles/lib_darktable.dir/develop/blend_gui.c.obj] Error 1
make[1]: *** [CMakeFiles/Makefile2:2250: bin/CMakeFiles/lib_darktable.dir/all] Error 2
make: *** [Makefile:171: all] Error 2

**Platform (please complete the following information):**
 - Darktable Version: 3.5.0+80
 - OS: Windows10/MinGW64

--- a/src/common/math.h
+++ b/src/common/math.h
@@ -21,6 +21,18 @@
 #include <math.h>
 #include <stdint.h>
 
+// work around missing standard math.h symbols
+/** ln(10) */
+#ifndef M_LN10
+#define M_LN10 2.30258509299404568402
+#endif /* !M_LN10 */
+
+/** PI */
+#ifndef M_PI
+#define M_PI   3.14159265358979323846
+#endif /* !M_PI */
+
+
 #define DT_M_PI_F (3.14159265358979324f)
 #define DT_M_PI (3.14159265358979324)
 

--- a/src/develop/blend_gui.c
+++ b/src/develop/blend_gui.c
@@ -19,6 +19,7 @@
 #include "bauhaus/bauhaus.h"
 #include "common/debug.h"
 #include "common/dtpthread.h"
+#include "common/math.h"
 #include "common/opencl.h"
 #include "common/iop_profile.h"
 #include "control/control.h"
@@ -35,7 +36,6 @@
 
 #include <assert.h>
 #include <gmodule.h>
-#include <math.h>
 #include <stdlib.h>
 #include <string.h>
 #include <strings.h>

--- a/src/iop/ashift_lsd.c
+++ b/src/iop/ashift_lsd.c
@@ -140,20 +140,9 @@
 
 #include <stdio.h>
 #include <stdlib.h>
-#include <math.h>
 #include <limits.h>
 #include <float.h>
-//#include "lsd.h"
-
-/** ln(10) */
-#ifndef M_LN10
-#define M_LN10 2.30258509299404568402
-#endif /* !M_LN10 */
-
-/** PI */
-#ifndef M_PI
-#define M_PI   3.14159265358979323846
-#endif /* !M_PI */
+#include "common/math.h"
 
 #ifndef FALSE
 #define FALSE 0

